qt/python/mantidqt/widgets/saveprojectdialog/presenter.py

```diff
@@ -0,0 +1,47 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench
+
+from qtpy.QtWidgets import QDialogButtonBox
+
+from mantidqt.widgets.saveprojectdialog.view import ProjectSaveDialogView
+from workbench.config import CONF
```

Comment:
I'm afraid we can't `import workbench` from `mantidqt`. `mantidqt` is meant to be the building blocks that the workbench application is built with so the dependency should be one way: `workbench->mantidqt` and not vice versa.
I think this would be better done with dependency injection. The `__init__` could take an additional optional parameter, `conf`, that if present is called to save the value. The workbench can then pass in the instance it uses and the unit test can pass in a mock object.